src/main/java/uk/gov/hmcts/reform/et.syaapi/config/OpenAPIConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.et.syaapi.config;
+
+import io.swagger.v3.oas.models.ExternalDocumentation;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenAPIConfiguration {
+
+    @Bean
+    public OpenAPI openAPI() {
+        return new OpenAPI()
+            .info(new Info().title("rpe demo")
+                      .description("rpe demo")
+                      .version("v0.0.1")
+                      .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
+            .externalDocs(new ExternalDocumentation()
+                              .description("README")
+                              .url("https://github.com/hmcts/spring-boot-template"));
+    }
```

Comment:
Is this accurate for et-sya-api?

Reply:
This is a copy of the OpenAPIConfiguration class from the Spring Boot template repo.
I can delete or update those properties.

Reply:
Could you please update it? It doesn't match our project.